src/utils/keyHelpers/accessData.ts

```diff
@@ -68,25 +42,7 @@ export async function getKeyIfExists (webId: NamedNode, keyUrl: string, keyType:
     const key = store.any(webId, ns.solid(keyType)) // store.sym(CERT + keyType))
     return key?.value // as NamedNode
   } catch (err) {
-    if (err.response.status === 404) {
```

Comment:
@bourgeoa I took out this code because it only creates the document but it doesn't create a key, I believe we take care of this a few methods above in the call stack. Let me know what you think. I can always put it back.